packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -113,10 +117,16 @@ class Shipping extends Component<ShippingProps & WithCheckoutShippingProps, Ship
         } catch (error) {
             onUnhandledError(error);
         } finally {
-            this.setState({ isInitializing: false });
+            if (this.isShippingMounted) {
```

Comment:
Do we need to add a test for this?

Reply:
I tried...but I didn't figure out how to access `isShippingMounted` in our context? 😂 Also, I didn't find we were testing `setState` before.